src/main/java/hellos/Hellos.java

```diff
@@ -13,12 +13,17 @@ public String generateOutput() {
     builder.append(jadenSaysHello());
     builder.append(kkSaysHello());
     builder.append(KenSaysHello());
+    builder.append(malenaSaysHello());
     builder.append(nicSaysHello());
     builder.append(patSaysHello());
 
     return builder.toString();
   }
 
+
+  private String malenaSaysHello() {
+    return "Malena says 'Hi!'\n";
+ }
```

Comment:
It looks like this close brace isn't indented correctly and should be one more space to the right?

Reply:
I would also typically put a blank line between function definitions, so add a blank line after this curly brace.